### DISCUSSION TITLE
unable to install using poetry

### DISCUSSION THREAD

**Discussion by fahadshery** [fastapi/sqlmodel#682] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
poetry add sqlmodel
Using version ^0.0.10 for sqlmodel

Updating dependencies
Resolving dependencies... (0.0s)

Because no versions of sqlmodel match >0.0.10,<0.0.11
 and sqlmodel (0.0.10) depends on pydantic (>=1.9.0,<2.0.0), sqlmodel (>=0.0.10,<0.0.11) requires pydantic (>=1.9.0,<2.0.0).
So, because fastapi-app2 depends on both pydantic (^2.4.2) and sqlmodel (^0.0.10), version solving failed.
```

### Description

unable to install using pottery...here is the poetry toml file:

```[tool.poetry]
name = "fastapi-app2"
version = "0.1.0"
description = "test"
authors =
license = "MIT"
readme = "README.md"

[tool.poetry.dependencies]
python = "^3.10"
uvicorn = "^0.23.2"
fastapi = "^0.104.0"
black = "^23.10.1"
pydantic = "^2.4.2"
alembic = "^1.12.1"
sqlalchemy = "^2.0.22"
email-validator = "^2.1.0.post1"


[build-system]
requires = ["poetry-core"]
build-backend = "poetry.core.masonry.api"
```

### Operating System

Linux

### SQLModel Version

can't show because not installed

### Python Version

Python 3.10.12

**Comment by seapagan** [comment]
You have pinned the latest version of Pydantic in your `pyproject.toml` (2.4.2). However, SQLModel needs a version less than 2.0. Below is the error you posted: 

```python
and sqlmodel (0.0.10) depends on pydantic (>=1.9.0,<2.0.0),
```

Pin your pydantic to 1.9 instead, 1.9.2 is the last 1.9 release from memory. I'm not sure if the 1.10.x branch works

```console
$ poetry add pydantic@1.9.2
```
